contracts/JetStaking/JetStakingV3.sol

```diff
@@ -72,7 +56,7 @@ contract JetStakingV3 is JetStakingV2 {
         uint256 latestRps = getLatestRewardPerShare(streamId);
         User storage userAccount = users[account];
         uint256 userRps = userAccount.rpsDuringLastClaim[streamId];
-        uint256 userShares = oldWeightedShareStreams[streamId]
+        uint256 userShares = streamId <= LAST_STREAM_ID
```

Comment:
should add `streamId != 0` check here too

Reply:
?